app/components/elements/forms/select_date_field_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div>
   <%= render Elements::Forms::LabelComponent.new(form:, field_name:, label_text: label, hidden_label:) %>
   <%= render Elements::Forms::HelpTextComponent.new(id: help_text_id, help_text:) %>
-  <%= select_day(value, { prompt: label, field_name:, prefix: }, { class: classes, aria: field_aria, data: }) %>
+  <%= select_unit(value, { prompt: label, field_name:, prefix: }, { class: classes, aria: field_aria, data: }) %>
```

Comment:
Collapse the select day/month stuff into a single reusable date helpers component.